### ISSUE TITLE
SWL relies on ZK for map info

### ISSUE THREAD

**Issue by ForbodingAngel** [springweblobby/swl-website#43]
This fails for obvious reasons. ZK fails at getting map or game data, gg no re.

**Comment by db81** [comment]
Yeah, it would be good to have a fallback. Right now swl only uses ZK infra for:
 * Map thumbnails, because you can [get a direct link from the map name](https://github.com/springweblobby/swl-website/blob/master/store/GameInfoImpl.js#L24) and I'd like to avoid having to do ajax queries to springfiles to get thumbnails.
 * Map search. IIRC springfiles limits the number of results which is a point against it but otherwise it's fine.

The big minimap image shown in battle screen is obtained from springfiles, zk ones are squished vertically and are lower res.

**Comment by abma** [comment]
> Map search. IIRC springfiles limits the number of results which is a point against it but otherwise it's fine.

only as default. append &limit=0 to the search url / as parameter and it returns all results.

this info isn't spread around as this shouldn't be used / only used very wisely as the full list is pretty large.

**Comment by db81** [comment]
Is there an offset parameter too to allow querying the list in small portions?

**Comment by abma** [comment]
not yet, thats a feature request (but should be easy to implement)
